### PR TITLE
Call EdpSimulator code for sampling VIDs

### PR DESCRIPTION
**Pull request by matthewclegg**
This change is [Reviewable](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement/474)

### CODE REVIEW

**Review by SanjayVas — APPROVED** [review]
> Reviewed 2 of 2 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement/474)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @kungfucraig)

**Review by matthewclegg — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement/474)* status: 0 of 2 files reviewed, 1 unresolved discussion (waiting on @kungfucraig)

___
*[src/main/kotlin/org/wfanet/measurement/loadtest/dataprovider/EdpSimulator.kt, line 209 at r1](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement/474#-MxkpkGeN0M5_7SoamVb-r1-209:-My9hcjd3G0rZkyXZrtj:bsxrlyc) ([raw file](https://github.com/world-federation-of-advertisers/cross-media-measurement/blob/b47a5273898251c86e455899992bc78eb82f383d/src/main/kotlin/org/wfanet/measurement/loadtest/dataprovider/EdpSimulator.kt#L209)):*
<details><summary><i>Previously, kungfucraig (Craig Wright) wrote…</i></summary><blockquote>

Given that any sketch can be configured to use an arbitrary fingerprinter (i.e. SHA256 or Farmhash), I'm a little nervous about this. That said, I don't think we need to change it, but maybe leave a comment? This could be the cause a fairly difficult to fix bug in the future.
</blockquote></details>

I added a comment to this effect.

File: src/main/kotlin/org/wfanet/measurement/loadtest/dataprovider/EdpSimulator.kt [hunk 1]
```diff
@@ -199,8 +206,12 @@ class EdpSimulator(
         socialGrade = SocialGrade.ABC1,
         complete = Complete.COMPLETE
       )
+    val vidSampler = VidSampler(Hashing.farmHashFingerprint64())
```
[the remark this answers is not in the document]
Comment (kungfucraig) [reply src/main/kotlin/org/wfanet/measurement/loadtest/dataprovider/EdpSimulator.kt:213]: Given your comments, do we need to salt the fingerprinter?

**Review by matthewclegg — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement/474)* status: 0 of 2 files reviewed, 1 unresolved discussion (waiting on @kungfucraig)

___
*[src/main/kotlin/org/wfanet/measurement/loadtest/dataprovider/EdpSimulator.kt, line 209 at r1](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement/474#-MxkpkGeN0M5_7SoamVb-r1-209:-My9luqv388Vnr6Oia-_:brtp0yg) ([raw file](https://github.com/world-federation-of-advertisers/cross-media-measurement/blob/b47a5273898251c86e455899992bc78eb82f383d/src/main/kotlin/org/wfanet/measurement/loadtest/dataprovider/EdpSimulator.kt#L209)):*
<details><summary><i>Previously, kungfucraig (Craig Wright) wrote…</i></summary><blockquote>

Given your comments, do we need to salt the fingerprinter?
</blockquote></details>

I suppose that depends on how you define 'need', not to mention 'we'. Kidding aside, let me propose three solutions in order of complexity.  Let me know if any of these appeal to you or if you would prefer something else:

1. Add more text to the comment suggesting salting of the fingerprinter.
2. Define some internal constant in this file, say VID_SAMPLE_SALT, that is added to each vid in the call to vidSampler.vidIsInSamplingBucket.
3. Submit a new PR modifying the VidSampler class to take an optional salt parameter in its constructor, which it would then add to each vid before calling the fingerprinter.

**Review by kungfucraig — APPROVED** [review]
> Reviewed 1 of 2 files at r1, 1 of 1 files at r3.
> *[Reviewable](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement/474)* status: all files reviewed (commit messages unreviewed), 1 unresolved discussion (waiting on @matthewclegg)

**Review by matthewclegg — COMMENTED** [review]
> Dismissed @kungfucraig from a discussion.
> *[Reviewable](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement/474)* status: all files reviewed (commit messages unreviewed), 1 unresolved discussion (waiting on @kungfucraig)

___
*[src/main/kotlin/org/wfanet/measurement/loadtest/dataprovider/EdpSimulator.kt, line 209 at r1](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement/474#-MxkpkGeN0M5_7SoamVb-r1-209:-MyDrnZ7BpOiZoU-bImv:b-x4dm1t) ([raw file](https://github.com/world-federation-of-advertisers/cross-media-measurement/blob/b47a5273898251c86e455899992bc78eb82f383d/src/main/kotlin/org/wfanet/measurement/loadtest/dataprovider/EdpSimulator.kt#L209)):*
<details><summary><i>Previously, kungfucraig (Craig Wright) wrote…</i></summary><blockquote>

We're in a situation where in production where Farmhash will be used by both AnySketch and the Sampler. Arguably we put the salt into the VidSampler class so that in production we ensure a common salt across implementers that is different from AnySketch, which we should embed a different salt into. So I guess #3 and some additional (perhaps optional) work on AnySketch would be my preference.

Given that, can you drop the comment in this PR, submit it, and then follow-up with PR on VidSampler?
</blockquote></details>

Yes, will do.

**Review by matthewclegg — COMMENTED** [review]
> Reviewed all commit messages.
> *[Reviewable](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement/474)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @matthewclegg)

**Pull request merged**